test/integration/traversal/TraversalTest.java

```diff
@@ -76,6 +86,143 @@ public void teardown() {
         databaseMgr.close();
     }
 
+    @Test
+    public void native_sorting_longs() {
```

Comment:
What part of these actually tests that the sorting is performed "natively"? Or do you just test that native sorting didn't break anything? In that case I'd remove `native` from the test names.